VideoWeb/VideoWeb/ClientApp/src/app/services/events.service.ts

```diff
@@ -186,6 +189,15 @@ export class EventsService {
             this.hearingTransferSubject.next(payload);
         });
 
+        this.connection.on(
```

Comment:
just confirming there should be no scenario where a judge would not be in the hearing before participants and need to send a message to the hearing `group` to force the participants to `sendMediaStatus` ?

Reply:
i think maybe we do given the below